packages/scripts/src/upgrade-tests/sherlock-update/0_Deploy.ts

```diff
@@ -0,0 +1,27 @@
+// This script deploys the OriginValidator.
+// This script can be run against a mainnet fork by spinning a node in a separate terminal with:
+// HARDHAT_CHAIN_ID=1 yarn hardhat node --fork https://mainnet.infura.io/v3/<YOUR-INFURA-KEY> --port 9545 --no-deploy
+// and then running this script with:
+// yarn hardhat run ./src/upgrade-tests/sherlock-update/0_Deploy.ts --network localhost
+
+const hre = require("hardhat");
+const { getContractFactory } = hre.ethers;
+
+async function main() {
+  console.log("DEPLOYING ORIGIN VALIDATOR");
+
+  const originValidator = await getContractFactory("OriginValidator");
+  const originValidatorContract = await originValidator.deploy();
```

Comment:
I can add this contract to `1.json` and verify it, it could be useful for future atomic proposals executions.